### PR TITLE
Upgrading to Swift 4.2

### PR DESCRIPTION
**Pull request by jreyesv**
### What
Upgrading to Swift 4.2 

### How
Running Xcode wizard and replacing use of `hidden` property with `isHidden`

### CODE REVIEW

File: stackableTests/HStackTests.swift [hunk 1]
```diff
@@ -52,7 +52,7 @@ class HStackTests: XCTestCase {
         let view2 = UIView()
         let size2 = CGSize(width: 55, height: 11)
         
-        let stack = HStack(spacing: spacing, layoutMargins: UIEdgeInsetsMake(topMargin, leftMargin, bottomMargin, rightMargin), thingsToStack: [
+        let stack = HStack(spacing: spacing, layoutMargins: UIEdgeInsets.init(top: topMargin, left: leftMargin, bottom: bottomMargin, right: rightMargin), thingsToStack: [
```
Comment (ronnychan-seek) [inline stackableTests/HStackTests.swift]: is `.init` necessary?

File: stackableTests/StackTests.swift [hunk 2]
```diff
@@ -69,7 +69,7 @@ class StackTests: XCTestCase {
         let rightMargin: CGFloat = 8
         let bottomMargin: CGFloat = 10
         
-        let stack = MockStack(layoutMargins: UIEdgeInsetsMake(topMargin, leftMargin, bottomMargin, rightMargin), thingsToStack: [ label1, MockStack(thingsToStack: [ label2 ]), label3 ])
+        let stack = MockStack(layoutMargins: UIEdgeInsets.init(top: topMargin, left: leftMargin, bottom: bottomMargin, right: rightMargin), thingsToStack: [ label1, MockStack(thingsToStack: [ label2 ]), label3 ])
```
Comment (ronnychan-seek) [inline stackableTests/StackTests.swift]: same as above

File: stackableTests/UILabel+heightForWidth+Tests.swift [hunk 3]
```diff
@@ -7,7 +7,7 @@ import XCTest
 
 class UILabel_heightForWidth_Tests: XCTestCase {
     func test_heightForWidth_should_return_expected_result() {
-        let font = UIFont.systemFont(ofSize: 12, weight: UIFontWeightMedium)
+        let font = UIFont.systemFont(ofSize: 12, weight: UIFont.Weight.medium)
```
Comment (ronnychan-seek) [inline stackableTests/UILabel+heightForWidth+Tests.swift]: might be able to use just `.medium`

File: stackableTests/VStackTests.swift [hunk 4]
```diff
@@ -52,7 +52,7 @@ class VStackTests: XCTestCase {
         let view2 = UIView()
         let height2: CGFloat = 11
         
-        let stack = VStack(spacing: spacing, layoutMargins: UIEdgeInsetsMake(topMargin, leftMargin, bottomMargin, rightMargin), thingsToStack: [
+        let stack = VStack(spacing: spacing, layoutMargins: UIEdgeInsets.init(top: topMargin, left: leftMargin, bottom: bottomMargin, right: rightMargin), thingsToStack: [
```
Comment (ronnychan-seek) [inline stackableTests/VStackTests.swift]: same as above

**Review by ronnychan-seek — COMMENTED** [review]
nice work @jreyesv, some minor questions/suggestions

File: stackableTests/HStackTests.swift (hunk 1 above)
Comment (jreyesv) [reply stackableTests/HStackTests.swift]: Good point, I will have a look

**Review by ronnychan-seek — APPROVED** [review]

**Pull request merged**